.pre-commit-config.yaml

```
@@ -63,6 +47,12 @@ repos:
           - rstcheck
           - sphinx
 
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.1.13
```

```suggestion
    rev: v0.1.14
```
And maybe place this block above, in place of `black`?

---

The ordering wasn't right, thank you!